force_nevergrad/mco/ng_mco_communicator.py

```diff
@@ -0,0 +1,90 @@
+#  (C) Copyright 2010-2020 Enthought, Inc., Austin, TX
+#  All rights reserved.
+
+import sys
+import re
+
+from force_bdss.api import (
+    BaseMCOCommunicator,
+    DataValue,
+    FixedMCOParameter,
+    RangedMCOParameter,
+    RangedVectorMCOParameter,
+    ListedMCOParameter,
+    CategoricalMCOParameter
+)
+
+
+class NevergradMCOCommunicator(BaseMCOCommunicator):
+    """ Command-line evaluation.
+
+    Examples
+    --------
+    Evaluate the point (-1.0, 1.0) and write to output.txt,
+
+    echo 1.0,-1.0 | edm run -e force-py36 -- force_bdss
+    --evaluate gaussian.json > output.txt
+
+    Evaluate the point on the first line of input.txt and write to output.txt,
+
+    Windows-native:
+    type input.txt | edm run -e force-py36 -- force_bdss
+    --evaluate gaussian.json > output.txt
+
+    Mac/bash:
+    cat input.txt | edm run -e force-py36 -- force_bdss
+    --evaluate gaussian.json > output.txt
```

Comment:
We can remove the `edm` elements from these commands - they could always be performed from within any environment that has the relevant installation packages. Additionally, we may end up changing the name of our development environment (i.e. migration to python 3.8 would mean `force-py38`), so we don't want to have to update lots of documentation if that happens.

```suggestion
    echo 1.0,-1.0 | force_bdss --evaluate gaussian.json > output.txt

    Evaluate the point on the first line of input.txt and write to output.txt,

    Windows-native:
    type input.txt | force_bdss --evaluate gaussian.json > output.txt

    Mac/bash:
    cat input.txt | force_bdss --evaluate gaussian.json > output.txt
```